src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
-    public int getUserScore(String[] records, String email) {
-        return 0;
+    public int getUserScore(String[] records, String email) throws UserNotFoundException {
+        for (int i = 0; i < records.length; i++) {
+            if (records[i].substring(0, records[i].indexOf(":")).equals(email)) {
+                return Integer.parseInt(records[i].substring(records[i].indexOf(":") + 1));
```

Comment:
let's split the record by ":" and save the result to the array. And then we can use data from this array)

Reply:
done